containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx

```diff
@@ -0,0 +1,82 @@
+---
+meta:
+  title: Moving Kubernetes nodes to routed IPs
+  description: Safely moving Kubernetes nodes to routed IPs
+content:
+  h1: Moving Kubernetes nodes to routed IPs
+  paragraph: Safely moving Kubernetes nodes to routed IPs
+tags: routed-ip ip-mobility kubernetes kapsule kosmos
+dates:
+  validation: 2023-04-22
+  posted: 2023-04-22
+categories:
+  - kubernetes
+---
+
+As part of our ongoing efforts to enhance infrastructure capabilities and provide better services to our customers, Scaleway is [introducing routed IPs for all products](https://www.scaleway.com/en/news/routed-ips-are-coming-to-all-scaleway-products/), including Kubernetes (K8s) worker nodes.
+
+One of the standout benefits of routed IPs is their support for [IP mobility](/compute/instances/concepts/#ip-mobility), streamlining IP management and movement within the Scaleway ecosystem. By simplifying the process of reallocating IPs across different products, routed IPs empower users with unprecedented flexibility and control over their network infrastructure.
+
+Additionally, routed IPs unlock the ability to attach multiple IPv4 addresses to a single Instance, enabling specific functionalities such as IP failover. While this feature may have limited applicability within Kubernetes environments, it offers invaluable flexibility and resilience across other Scaleway products.
+
+<Message type="note">
+  Routed IPs provide primary support for IPv6, ensuring direct connectivity for IPv6 addresses with Instances.
+  However, IPv6 support for Kapsule is not yet available due to compatibility limitations with existing Container Network Interfaces (CNIs).
+</Message>
+
+<Message type="important">
+    NAT IPs will reach end-of-life in Q4 2024. After this date, Scaleway will organize migrations for all clusters to transition to routed IPs.
+    Migrations will be planned region by region, with specific dates to be announced.
+</Message>
+
+## Moving your Kubernetes nodes to routed IPs
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Set up a [Kubernetes Kapsule cluster](/containers/kubernetes/how-to/create-cluster)
+
+<Message type="note">
+  Newly created Kapsule clusters are deployed with IP mobility enabled by default. For existing clusters, you can manually launch the migration to routed IPs.
```

Comment:
Kapsule and Kosmos clusters created after April 12th 2024 are deployed with IP mobility enabled by default ([changelog](https://www.scaleway.com/en/docs/changelog/?product=kubernetes#routed-ips-are-now-default-on-kubernetes-clusters))

Reply:
```suggestion
  Kapsule and Kosmos clusters created after April 12th 2024 are deployed with IP mobility enabled by default. For existing clusters, you can manually launch the migration to routed IPs.
```